subsys/zigbee/common/zigbee_helpers.c

```diff
@@ -11,21 +11,27 @@
 
 #include <dk_buttons_and_leds.h>
 #include <logging/log.h>
+#include <hal/nrf_power.h>
```

Comment:
Do you know if there are plans to have RAM power management in a platform independent Zephyr's API?

Reply:
Unfortunately, haven't heard of such a feature